shared-bindings/displayio/Display.c

```diff
@@ -246,8 +250,19 @@ STATIC mp_obj_t displayio_display_obj_refresh(size_t n_args, const mp_obj_t *pos
     if (minimum_frames_per_second > 0) {
         maximum_ms_per_real_frame = 1000 / minimum_frames_per_second;
     }
-    return mp_obj_new_bool(common_hal_displayio_display_refresh(self, 1000 / args[ARG_target_frames_per_second].u_int, maximum_ms_per_real_frame));
+
+    uint32_t target_ms_per_frame;
+    if ( (args[ARG_target_frames_per_second].u_obj == mp_const_none) || (n_args == 1) ) {
```

Comment:
```suggestion
    if (args[ARG_target_frames_per_second].u_obj == mp_const_none) {
```

You can simplify this by setting it to `mp_const_none` by default. Please also update the documentation comment above. The type will be `Optional[int]` with the value `None`.

Reply:
I don't know how most folks use `refresh`, but currently it can be used solely by setting the `minimum_frames_per_second` to a given value and the `target_frames_per_second` will default to 60. 

I wrote this version to closely replicate the current version, so it would default to `target_frames_per_second` value of 60, and the only change is to accept the `None` value, or called with no parameters at all to trigger the immediate refresh.

Before making the change and updating the docs, I want to clarify that you want to change how this function operates.

Reply:
I made a commit to include the changes requested, including the update to the documentation..